charts/mentoring/values.yaml

```diff
@@ -0,0 +1,69 @@
+# Default values for mentoring.
+replicaCount: 1
+
+image:
+  repository: 783633885093.dkr.ecr.us-west-2.amazonaws.com/mentoring
+  pullPolicy: IfNotPresent
+  tag: "0.0.1"
+
+fullnameOverride: "mentoring"
+
+serviceAccount:
+  create: true
+
+service:
+  type: ClusterIP
+  port: 80
+
+ingress:
+  enabled: true
+  annotations:
+    kubernetes.io/ingress.class: nginx-mentoring
+  hosts:
+    - host: mentoring.mozilla.com
+      paths:
+        - path: /
+      backend:
+        serviceName: mentoring
+        servicePort: 80
+
+resources:
+  limits:
+    cpu: 500m
+    memory: 1024Mi
+  requests:
+    cpu: 100m
+    memory: 128Mi
+
+autoscaling:
+  enabled: true
+  minReplicas: 1
+  maxReplicas: 3
+  targetCPUUtilizationPercentage: 80
```

Comment:
This seems a bit high to expect a typical django app to reach under load. I'd worry we'd spend more than 20% of time in DB.

Reply:
sure, that make sense. Which threshold would you propose?

Reply:
Maybe 60, but i'll defer to @djmitche if they have opinions

Reply:
I'm not sure what this means.  What's the value for the moderator app?

Reply:
This value controls when to scale the amount of replicas of the application. Currently it says: scale the amount of containers when the CPU utilization (avg of 5 mins) is above 80%. Min replicas is 1, max is 3.

It's unlikely the application will scale often or even at all, because we are giving it 0.5 cores and 1GB of RAM.
Anyway it's always good to have autoscaling on.

So here the question would be, what would be a correct threshold to scale the number of replicas.

Reply:
Moderator is using that 80% value too.

Reply:
Since this app shouldn't need to scale and has little impact if it is unavailable, I think we can forget I raised this.

Reply:
Same, and "whatever moderator does" is probably a good rule of thumb :)

Reply:
perfect. In that case I'll leave it as it is. Thank you for the discussion